SAPHana/ra/SAPHanaTopology

```diff
@@ -893,6 +893,10 @@ function sht_monitor_clone() {
        vName=${NODENAME}
```

Comment:
In line 380 the changes from your pull request 41 are missing. If we now merge this PR, we will lost the change. Can you please check.

Reply:
No, that was my fault. As we do not have overlapping changes, we will not lost the changes. Sorry for the noise.

Reply:
Unfortunately I had the need to use different branches. Will fix that.